src/lighteval/few_shot_manager.py

```diff
@@ -181,35 +182,33 @@ def init_fewshot_sampling_balanced(
     def get_examples_with_chat_template(
```

Comment:
I had to change this method to return `List[ChatCompletionInputMessage]` as `InferenceClient.chat_completion()` doesn't accept string. I made changes accordingly to `BaseModel` and `NanotronModel` to consider conversational contexts.

Reply:
This comment now is relevant to `PromptManager.get_examples()`.